docs/architecture-decisions/2024-03-06_local_dev_ui_packaging.md

```diff
@@ -0,0 +1,62 @@
+# Local dev UI packaging
+
+- **Status**: Draft
+- **Owner:** Patrick Dinh (MakerX)
+- **Deciders**: Alessandro (Algorand Foundation), Rob Moore (MakerX), MakerX team
+- **Date created**: 2024-03-06
+
+## Context
+
+We are building a web-based local explorer interface to support:
+
+- Transaction / asset / account explorer
+- Dev wallet integration (including KMD support)
+- Network switching
+- ARC-32 app studio
+- Visual transaction representations and basic visual debugging interface to help new users understand what’s happening
+- AlgoKit init and Algokit tasks CLI exploration user interfaces to help new users get the most out of the CLI
+
+The explorer will be packaged as a executable as well as deployed as a website.
+
+## Requirements
+
+- The explorer should support a wide variety of Linux distributions, macOS (both Apple Silicon and Intel architectures), and Windows.
```

Comment:
Probably make sense to include something about filesystem access to faciliate more powerful integration to algokit projects.